ghost/core/core/server/services/mega/email-preview.js

```diff
@@ -36,7 +36,11 @@ class EmailPreview {
             );
         });
 
-        return emailContent;
+        return {
```

Comment:
We explicitly return specific keys instead of returning the object as-is which can include extra bits like `post` that we don't want to return.